website/docs/validating-admission-policy.md

```diff
@@ -54,22 +58,9 @@ Find out more about different [enforcement points](enforcement-points.md)
     runtimeConfig:
       admissionregistration.k8s.io/v1beta1: true
     ```
-- Set `--experimental-enable-k8s-native-validation` in Gatekeeper deployments, or `enableK8sNativeValidation=true` if using Helm.
```

Comment:
we should mention set to false if you want to opt-out

Reply:
Mentioned setting false to opt-out with the note above.